CMakeLists.txt

```diff
@@ -324,64 +414,73 @@ if(NOT BUILD_SHARED_LIBS)
   endif()
 endif()
 
-# Detect H5Part
+# H5Part
+if(ENABLE_AMPI)
+  set(H5PART_ROOT ${CMAKE_INSTALL_PREFIX})
+endif()
 find_package(H5Part)
-if(H5Part_FOUND)
+
+if ((ENABLE_AMPI AND H5Part_FOUND AND
+     H5PART_INCLUDE_DIRS MATCHES ${CMAKE_INSTALL_PREFIX}) OR
+    (NOT ENABLE_AMPI AND H5Part_FOUND))
   set(h5part "")
 else()
   set(h5part "h5part")
+  message(STATUS "H5Part either not found or AMPI is enabled: will build")
 endif()
 
-# Detect TestU01
+# TestU01
 find_package(TestU01)
 if(TestU01_FOUND)
   set(testu01 "")
 else()
   set(testu01 "testu01")
 endif()
 
-# Detect Trilinos system-wide and on paths given by "<package>_DIR" cmake or
-# environment variables.
+# Trilinos
 find_package(Zoltan2 QUIET)
 find_package(SEACASExodus QUIET)
 find_package(Exodiff QUIET)
-if(Zoltan2_FOUND AND SEACASExodus_FOUND AND Exodiff_FOUND)
+
+if ((ENABLE_AMPI AND Zoltan2_FOUND AND SEACASExodus_FOUND AND Exodiff_FOUND AND
+     Zoltan2_INCLUDE_DIRS MATCHES ${CMAKE_INSTALL_PREFIX}) OR
+    (NOT ENABLE_AMPI AND Zoltan2_FOUND AND SEACASExodus_FOUND AND Exodiff_FOUND))
   set(trilinos "")
 else()
   set(trilinos "trilinos")
+  message(STATUS "Zoltan2, SEACASExodus, and Exodiff either not found or AMPI is enabled: will build")
 endif()
 
-# Detect BoostMPLCartesianProduct
+# BoostMPLCartesianProduct
 find_package(CartesianProduct)
 if(CARTESIANPRODUCT_FOUND)
   set(cartesian_product "")
 else()
   set(cartesian_product "cartesian_product")
 endif()
 
-# Detect Template Unit Test library
+# Template Unit Test library
 find_package(TUT)
 if(TUT_FOUND)
   set(tut "")
 else()
   set(tut "tut")
 endif()
 
-# Detect Zlib (only for static link and only on ppc64, required because we do
-# not build it if it is not found)
+# Zlib, only for static link and only on ppc64
 if(ARCH MATCHES "ppc64")
  find_package(ZLIB REQUIRED)
 endif()
 
-# Detect NumDiff
+# NumDiff
```

Comment:
No, these comments become really useless.

Reply:
Yeah, I agree, but at least now they are all _consistently_ useless. ;-)

Fixed the conflict - I hope.